src/app/features/create/components/hydrate-new-vehicle-record/hydrate-new-vehicle-record.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { GlobalError } from '@core/components/global-error/global-error.interface';
+import { GlobalErrorService } from '@core/components/global-error/global-error.service';
+import { Axle, VehicleTechRecordModel } from '@models/vehicle-tech-record.model';
+import { Actions, ofType } from '@ngrx/effects';
+import { Store } from '@ngrx/store';
+import { TechnicalRecordService } from '@services/technical-record/technical-record.service';
+import { createVehicleRecord, createVehicleRecordSuccess } from '@store/technical-records';
+import { TechnicalRecordServiceState } from '@store/technical-records/reducers/technical-record-service.reducer';
+import { take, Observable } from 'rxjs';
+
+@Component({
+  selector: 'app-hydrate-new-vehicle-record',
+  templateUrl: './hydrate-new-vehicle-record.component.html'
+})
+export class HydrateNewVehicleRecordComponent implements OnInit {
+  constructor(
+    private actions$: Actions,
+    private globalErrorService: GlobalErrorService,
+    private route: ActivatedRoute,
+    private router: Router,
+    private store: Store<TechnicalRecordServiceState>,
+    private technicalRecordService: TechnicalRecordService
+  ) {}
+
+  ngOnInit() {
+    !this.vehicle$.subscribe(data => {
```

Comment:
```suggestion
    this.vehicle$.subscribe(data => {
```
not sure what that's doing

Reply:
fixt